tools/swift-ide-test/swift-ide-test.cpp

```diff
@@ -3871,6 +3876,12 @@ int main(int argc, char *argv[]) {
         InitInvok.getLangOptions().EffectiveLanguageVersion = actual.getValue();
     }
   }
+  if (!options::ToolsDirectory.empty()) {
```

Comment:
I guess there's not really a good way to test this path...

Reply:
Yeah, and since this is only used by Swift tests I think it should be fine to leave as is.